### PR TITLE
feat[GITOPSRVCE-750]: Add visualization for Synced Applications SLO

### PR DESCRIPTION
**Pull request by keithchong**
See https://issues.redhat.com/browse/GITOPSRVCE-750

Screen Shot 2023-10-13 at 6 24 57 PM

### CODE REVIEW

**Comment by yftacherzog** [comment]
@keithchong gitlint seems to be misbehaving here. It somehow fails for the square brackets on the commit title. Can you edit the commit message and switch them with round brackets? i.e.
```
feat(GITOPSRVCE-750): Add visualization for Synced Applications SLO
...
```

**Comment by keithchong** [comment]
Commit message updated

File: dashboards/grafana-dashboard-rhtap-slos.configmap.yaml [hunk 1]
```diff
@@ -66,6 +65,174 @@ data:
...
+              "range": true,
+              "refId": "B"
+            },
+            {
+              "datasource": {
+                "name": "Expression",
+                "type": "__expr__",
+                "uid": "__expr__"
+              },
+              "expression": "$A / $B",
```
Comment (amisstea) [inline dashboards/grafana-dashboard-rhtap-slos.configmap.yaml:219]: This should probably be multiplied by 100 to make it an actual percentage.

Comment (keithchong) [reply dashboards/grafana-dashboard-rhtap-slos.configmap.yaml]: Done.

**Review by amisstea — APPROVED** [review]

**Review by hmariset — APPROVED** [review]
LGTM

**Comment by keithchong** [comment]
I had to rebase, another panel was added by the HAS team.

Here is the updated view. Note, the GitOps service team will put the SLO panels under the collapsible row "GitOps Service SLOs"

Screen Shot 2023-10-26 at 3 47 49 PM

**Pull request merged**